src/GraphODataTemplateWriter/CodeHelpers/MethodInfo.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+
+namespace Microsoft.Graph.ODataTemplateWriter.CodeHelpers
+{
+    /// <summary>
+    /// Contains the OData method information used in method request builders.
+    /// </summary>
+    public class MethodInfo
+    {
+        public MethodInfo(IOrderedEnumerable<ParameterInfo> parameters, 
```

Comment:
I don't think the constructor is adding value here, switch to property initialization and remove it?

Reply:
Agreed. Actually, it removes value since this model may contain options that are not required for other languages. Making this change.